src/components/graphs/states/Wrapper.tsx

```diff
@@ -37,9 +35,9 @@ function GraphStateWrapper({ children }: BaseComponentProps) {
                 }
             />
         );
-    } else {
-        return <GraphLoadingState />;
     }
+
+    return <GraphLoadingState />;
```

Comment:
Removed some of the `if else if` stuff so it can more easily default to loading and not manually check so much stuff.